.github/workflows/rapid-test.yml

```diff
@@ -3,58 +3,67 @@ name: Rapid Test
 on:
   push:
     branches: [master]
-    paths-ignore:
-      - '.github/workflows/docs.yml'
-      - '.markdownlintrc'
-      - '.remarkignore'
-      - '.remarkrc.mjs'
-      - '.retext-spell.dic'
-      - 'docs/**'
-      - '**.md'
+    paths:
```

Comment:
Using actually relevant paths — should decrease number of false alerts where Rapid Test phase is running due to unrelated changes.